### PR TITLE
M1 multipole in EPL class

### PR DESCRIPTION
**Pull request by dangilman**
I add a new lens profile class that includes a spherical m=1 multipole perturbation concentric with an EPL profile, an m=3 perturbation, and an m=4 perturbation

### CODE REVIEW

**Comment by dangilman** [comment]
@sibirrer @ajshajib  there seems to be an issue with the build, but the tests are passing, same issue affects another open pull request

**Comment by sibirrer** [comment]
> @sibirrer @ajshajib there seems to be an issue with the build, but the tests are passing, same issue affects another open pull request

yeah @dangilman, the pre-commit ci is broken. You can ignore that part

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+
+__all__ = ["EPL_MULTIPOLE_M1M3M4", "EPL_MULTIPOLE_M1M3M4_ELL"]
+
+
+class EPL_MULTIPOLE_M1M3M4_ELL(LensProfileBase):
+    """EPL (Elliptical Power Law) mass profile combined with two elliptical multipole
+    terms of order m=3 and m=4 (exact for general axis ratio q) and a circular m=1
+    multipole.
+
+    See also documentation of EPL_BOXYDIKSY CLASS, lenstronomy.LensModel.Profiles.epl
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:17]: are there references to a paper that uses this parameterization?

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+    and lenstrnomy.LensModel.Profiles.multipole for details.
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:18]: typo in lenstronomy

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+        delta_phi_m3,
+        a4_a,
+        delta_phi_m4,
+        center_x=0,
+        center_y=0,
+    ):
+        """Computes the derivatives of the potential (deflection angles)in units of
+        theta_E.
+
+        :param x: x-coordinate in image plane :param y: y-coordinate in image plane
```
Comment (sibirrer) [inline lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:245]: docstrings are duplicated

**Review by sibirrer — CHANGES_REQUESTED** [review]
Looks great! I have some very minor comments

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py (hunk 1 above)
Comment (dangilman) [reply lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:17]: not using elliptical multipoles, but there are some papers that use all three circular ones. I can add a reference

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py (hunk 2 above)
Comment (dangilman) [reply lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:18]: fixed

File: lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py (hunk 3 above)
Comment (dangilman) [reply lenstronomy/LensModel/Profiles/epl_multipole_m1m3m4.py:245]: fixed

**Pull request closed without merging**